server/get-browsers.js

```diff
@@ -0,0 +1,106 @@
+import browserslist from 'browserslist'
+import { readFileSync } from 'fs'
+import { URL } from 'url'
+import { agents as caniuseAgents, region as caniuseRegion } from 'caniuse-lite'
+
+let { version: bv } = importJSON('./node_modules/browserslist/package.json')
+let { version: cv } = importJSON('./node_modules/caniuse-lite/package.json')
+let wikipediaLinks = importJSON('../browsers-data/wikipedia-links.json')
+
+export default async function getBrowsers(query, region) {
+  // TODO Add support `Node > 0` query
+  let loadBrowsersData = async (resolve, reject) => {
+    let browsersByQuery = []
+
+    try {
+      browsersByQuery = browserslist(query)
+    } catch (error) {
+      reject(
+        error.browserslist
+          ? error.message
+          : `Unknown browser query \`${query}\`.`
+      )
+      return
+    }
+
+    let browsersGroups = {}
+    let browsersGroupsKeys = []
+
+    for (let browser of browsersByQuery) {
+      if (browsersGroupsKeys.includes(browser)) {
+        return
+      }
+
+      browsersGroupsKeys.push(browser)
+      let [id, version] = browser.split(' ')
+      let versionCoverage = region
+        ? await getRegionCoverage(id, version, region)
+        : getGlobalCoverage(id, version)
+
+      let versionData = [version, roundNumber(versionCoverage)]
```

Comment:
@ai 
```
[['101': 23]]
```

It's not too cryptic? :)

Reply:
`{ 101: 2.4, 102: 46.06 }` will be ideal.

Let's client round data and object is the simplest for user.